src/shims/unix/fd.rs

```diff
@@ -479,58 +480,64 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
     }
 
     fn fcntl(&mut self, args: &[OpTy<'tcx>]) -> InterpResult<'tcx, Scalar> {
+        let [fd_num, cmd] = check_min_arg_count("fnctl", args)?;
+
         let this = self.eval_context_mut();
 
```

Comment:
```suggestion
        let this = self.eval_context_mut();

        let [fd_num, cmd] = check_min_arg_count("fnctl", args)?;
```
`let this` always comes first, by convention

Reply:
Should then I change other places where I find it going after?